resources/assets/less/bem/btn-home.less

```
@@ -52,6 +52,10 @@
     --move: 2px;
   }
 
+  &--c-darkorange {
+    --colour-h: var(--colour-darkorange-hue)
```

the css (less) file is invalid due to missing semicolon
```suggestion
    --colour-h: var(--colour-darkorange-hue);
```